data.json

```diff
@@ -718,7 +718,49 @@
             {
                 "name": "webstorm-help-hub",
                 "description": "Get help with WebStorm",
-                "type": "forum"
+                "type": "forum",
+                "available_tags": [
```

Comment:
There is a limit of 20 max tags in a channel

Reply:
I can improve those and write some validations for it, as I will likely forgot those constraints in a day or two :D